client/packages/requisitions/src/RequestRequisition/DetailView/Toolbar/Toolbar.tsx

```diff
@@ -129,7 +129,7 @@ export const Toolbar: FC = () => {
         display="flex"
         gap={1}
         justifyContent="flex-end"
-        sx={{ marginTop: 2 }}
+        sx={{ marginTop: 2, height: 37 }}
```

Comment:
I'm too rusty on react but generally speaking adding random inline heights in the UI makes me sad 😢. Why does the search bar seemingly have padding above it? Or is it the DDL has margin below it? Ideally things would just grow to the container and the container aligns them :(

Reply:
Have removed and done with with some flexing instead 💪🏻